app/components/FactsPageContent.tsx

```diff
@@ -1,19 +1,29 @@
+'use client';
+
 import { Earthquakes } from '@/types';
 import StatsSection from './StatsSection/StatsSection';
+import useSyncAtom from '@/store/useSyncAtom';
+import { allDailyEventsAtom, dailyEventsWithTimesAtom } from '@/store';
 
 type FactsPageContent = {
+  allDailyEvents: any; // this is ALL the earthquakes for the day, regardless of magnitude
```

Comment:
This comment is no longer true - the daily events are now restricted to the "significant magnitude" of 2.5 or greater